spyder/plugins/plots/widgets/figurebrowser.py

```diff
@@ -212,6 +215,15 @@ def __init__(self, parent=None, background_color=None):
         self.stack_layout.setSpacing(0)
         self.setContentsMargins(0, 0, 0, 0)
 
+    def on_splitter_moved(self):
+        total_width = self.splitter.width()
+
+        min_width_percentage = 0.5
+
+        min_width = total_width * min_width_percentage
+
```

Comment:
```suggestion
    def _on_splitter_moved(self):
        total_width = self.splitter.width()
        min_width_percentage = 0.55
        min_width = int(total_width * min_width_percentage)
```

* Make method private.
* Remove unnecessary blank lines.
* Make `figviewer` slightly bigger than the thumbnails bar.
* `min_width` needs to be an `int` in Python 3.10+